### PR TITLE
(docs): for how to connect to UI on localhost 

### LINKED ISSUE #33: (docs) explain how to connect to UI
The readme in the compose file directories explain how to start the environment and then immediately jump to the  username/password.

The readme should explain how to start the UI. At this stage, I would explain it only for a local installation (http://localhost:8080). The explanation on how to proceed with GitPod can come later.

### PR DESCRIPTION
**Pull request by ash-sxn**
- fixes #33 
-  pinging mentors @berviantoleo @jmMeessen @gounthar

### CODE REVIEW

File: experimental_docker_compose_files/00_old_one_from_proposal/README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 ### How to run 
 * Make sure you are in the `00_old_one_from_proposal` directory by using the command `pwd`. 
 * Then use the command `docker compose up -d` to run the example.
+* after the containers are succefully running (check with `docker ps`). You can access the controller on [`http://localhost:8080`](http://localhost:8080)
```
Comment (gounthar) [inline experimental_docker_compose_files/00_old_one_from_proposal/README.md:9]:
```suggestion
* after the containers are successfully running (check with `docker compose ps`). You can access the controller on [`http://localhost:8080`](http://localhost:8080)
```

**Review by gounthar — CHANGES_REQUESTED** [review]

**Review by jmMeessen — APPROVED** [review]
We need to explain the special case of GitPod.
But at this stage the update to the readme is good enough. Needs to be revisited later (as well as the general documentation)

**Review by berviantoleo — APPROVED** [review]

**Review by gounthar — APPROVED** [review]

**Comment by jmMeessen** [comment]
Just tried it out on Gitpod. The URL to call is something in this form: https://8080-ashsxn-gsoc2023dockerba-74xsllkyjcw.ws-eu100.gitpod.io/
We just need to tool it in such a way that it is straightforward for the user to start a session.

**Comment by gounthar** [comment]
What about adding another bash script calling the existing key-generating script and another one that would detect the "machine" name under gitpod, and change the JCasc configuration accordingly?

**Comment by ash-sxn** [comment]
I was thinking for the gitpod part, since it's mostly for beginners we could create a [`.gitpod.yaml`](https://www.gitpod.io/docs/references/gitpod-yml), that does all this and both  ssh-key update and running `docker compose up` too, right now since this is in a subdirectory and there are other examples, we can implement that for our main example

**Comment by ash-sxn** [comment]
Isn't mentioning going to the `PORTS` section and clicking on the address a better solution for gitpod right now, and a popup also appears that leads to UI when the containers are built. We can implement @gounthar 's above script in the  .gipod.yml file for the final example, what do you guys think?

**Comment by gounthar** [comment]
> I was thinking for the gitpod part, since it's mostly for beginners we could create a [`.gitpod.yaml`](https://www.gitpod.io/docs/references/gitpod-yml), that does all this and both ssh-key update and running `docker compose up` too, right now since this is in a subdirectory and there are other examples, we can implement that for our main example

I would prefer having a single point of entry, a `bash` script that would detect whether we are on MacOs, Linux or GitPod, and act accordingly.
You already have the parts to detect MacOs (`"$OSTYPE" == "darwin"*`) and GitPod (`-n "$GITPOD_WORKSPACE_URL"`).

**Comment by gounthar** [comment]
> Isn't mentioning going to the `PORTS` section and clicking on the address a better solution for gitpod right now, and a popup also appears that leads to UI when the containers are built. We can implement @gounthar 's above script in the .gipod.yml file for the final example, what do you guys think? for updating URL part in jenkins.yaml I will add a script in #36. which we can later add to the `.gipod.yaml` file so it automatically updates jenkins.yaml file with correct URL when gitpod is started

I think it's a good idea to explain in the README how things work under GitPod. However, we still need to implement the JCasC modification part, where we will set the calculated value of the Jenkins controller instance as the default URL before starting the Docker Compose.

**Comment by ash-sxn** [comment]
Okay,

> > I was thinking for the gitpod part, since it's mostly for beginners we could create a [`.gitpod.yaml`](https://www.gitpod.io/docs/references/gitpod-yml), that does all this and both ssh-key update and running `docker compose up` too, right now since this is in a subdirectory and there are other examples, we can implement that for our main example
> 
> I would prefer having a single point of entry, a `bash` script that would detect whether we are on MacOs, Linux or GitPod, and act accordingly. You already have the parts to detect MacOs (`"$OSTYPE" == "darwin"*`) and GitPod (`-n "$GITPOD_WORKSPACE_URL"`).

For linux and MacOS we can set default value to `http://127.0.0.1:8080` and for gitpod we can have a script  but for `yq`, since it's not installed by default, we can put a .gitpod.yaml file that install the `yq` command when gitpod is launched

**Comment by ash-sxn** [comment]
> > I was thinking for the gitpod part, since it's mostly for beginners we could create a [`.gitpod.yaml`](https://www.gitpod.io/docs/references/gitpod-yml), that does all this and both ssh-key update and running `docker compose up` too, right now since this is in a subdirectory and there are other examples, we can implement that for our main example
> 
> I would prefer having a single point of entry, a `bash` script that would detect whether we are on MacOs, Linux or GitPod, and act accordingly. You already have the parts to detect MacOs (`"$OSTYPE" == "darwin"*`) and GitPod (`-n "$GITPOD_WORKSPACE_URL"`).

For Linux and macOS we can have `http://127.0.0.1:8080` as default and script for gitpod that changes that, but since it needs `yq`  command, I can make `.gitpod.yaml` file that installs `yq`  on gitpod

**Comment by gounthar** [comment]
> For Linux and macOS we can have `http://127.0.0.1:8080` as default and script for gitpod that changes that, but since it needs `yq` command, I can make `.gitpod.yaml` file that installs `yq` on gitpod
Fine with me! 👍

**Pull request merged**